chicago/static/css/custom.css

```diff
@@ -511,11 +511,6 @@ textarea {
   font-size: 0.75em;
 }
 
-.img-thumbnail {
```

Comment:
when we wrote this i think we didn't know what this `img-thumbnail` class was for. i restored its original markup, which adds a border